modules/recommended-practices-for-etcd-hosts.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/recommended-host-practices.adoc
+
+[id='recommended-practices-for-etcd-hosts-{context}']
+= Recommended practices for etcd hosts
```

Comment:
Will you confirm that there are still etcd hosts? I'm pretty sure that in 4.0 you can only run etcd as a static pod.

Reply:
There are no etcd hosts but we wanted to keep the information for reference even though it's not really relevant anymore

Reply:
If it's not really relevant, someone's going to get confused by it. I'd consider removing it.

Reply:
Yeah, it's not my work so I don't feel as bad about cutting it :disappointed:

Reply:
We're cutting a lot of content and rewriting most of the rest, so I don't feel bad about removing it.

Reply:
Axing it :hocho:  RIP irrelevant content :ghost: